ads/adop.md

```diff
@@ -0,0 +1,38 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Adop
+
+## Example
+
+### Basic
+
+```html
+<amp-ad width="300" height="250"
+     type="adop"
+     data-z="a3a8014f-0b5f-46dd-a0db-a2ae686541ef">
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact http://adop.cc
+
+### Required parameters
+
+- `data-z`
+
+### Optional parameters
```

Comment:
If there are no optional parameters, can we remove this header? 😄

Reply:
Sure. Thank you.